modules/installation-configuration-parameters.adoc

```diff
@@ -384,6 +384,16 @@ ifdef::gcp[]
 |The name of the existing VPC that you want to deploy your cluster to.
 |String.
 
+|`platform.gcp.type`
+|The link:https://cloud.google.com/compute/docs/machine-types[GCP machine type].
+|The subnet name.
+
+|`platform.gcp.zones`
+|The availability zones where the installation program creates machines for the
+control plane MachinePool.
```

Comment:
> control plane MachinePool
The zone is not only used for control plane machinepool, but also compute nodes machinepool